translator/trace/zipkin/zipkinv2_to_traces.go

```diff
@@ -164,6 +164,11 @@ func populateSpanStatus(tags map[string]string, status pdata.SpanStatus) {
 			delete(tags, tracetranslator.TagStatusMsg)
 		}
 	}
+
+	if _, ok := tags[tracetranslator.TagError]; ok {
```

Comment:
does the value matter? `true` or `false`?

Reply:
hmm, It looks like only lower case `true` is accepted, so i will update

ref:
- https://github.com/open-telemetry/opentelemetry-specification/pull/1257/files#diff-60d9dfa59a8a1a0776c260526b6148549d4087deb626d42732fdd7616ef7a636R56

- https://github.com/open-telemetry/opentelemetry-specification/pull/1257/files#diff-c50b2cac17490acd52955c504c267e15e9b8f1bfd51f15e02369cb1c83071e13R130

Reply:
Updated